tools/fixup-wheels.sh

```diff
@@ -82,10 +82,12 @@ fixup_wheel_macosx () {
 
         # Change the name to be local
         install_name_tool -id "$lib" $lib
-        if otool -L $lib | grep -q /usr/local/lib/librdkafka.1.dylib ; then
+        # Extract existing(old) reference
+        old=$(otool -L $lib | grep -o '.*librdkafka.1.dylib' | xargs)
```

Comment:
nit: Nice trick, but will not handle multiple spaces in the path itself (however unlikely).
```
$ echo '  y/    strange path/x' | xargs
y/ strange path/x
```

That's ok for now though.,